### ISSUE TITLE
Ignore false positives in code

### ISSUE THREAD

**Issue by KnisterPeter** [depcheck/depcheck#699]
### Feature request description

It would be nice to be able to mark false positives as such.
Like the eslint magic comments or typescript magic comments to suppress errors which are false positives. 

### Code snippets (if applicable)

How the dependency is used (if applicable):

```js
import webpack from 'webpack'; // depcheck-ignore
```

How the dependency is listed in package.json (if applicable):

```json
{
  "devDependencies": {
  }
}
```

`webpack` is imported, but not declared. There might be valid reasons to do so and it would be nice to suppress that error.
